playground/backend/internal/setup_tools/run_builder/run_builder_setuper.go

```diff
@@ -23,6 +23,8 @@ import (
 	"beam.apache.org/playground/backend/internal/logger"
 	"fmt"
 	"github.com/google/uuid"
+	"path/filepath"
+	"strings"
 )
 
 // Setup returns executors.RunBuilder setup it according to sdk
```

Comment:
keep in mind that this part was refactored by @daria-malkova

Reply:
I am trying =)